index.html

```diff
@@ -294,33 +294,33 @@ <h2 class="contact-h2">
                 </h2>
             </div>
             <form action="https://formspree.io/f/xnqrnjgr" class="contact-details" method="post">
```

Comment:
- [ ] Good job so far :+1:  But your project is not working as expected :cry:  Please make sure that your application is collecting data by using the [Formspree service](https://formspree.io/html) correctly because it is part of the project requirement. :+1: 
See the screenshot below :arrow_down: 

![Screenshot from 2022-08-30 17-54-09](https://user-images.githubusercontent.com/6615394/187511354-875f0b59-bb64-4b62-8830-1122028f92c8.png)